### ISSUE TITLE
Bullet points in the comments appear out to the left

### ISSUE THREAD

**Issue by 345k** [lbryio/lbry-desktop#3221]
**Bug**

When I visit a post that has a comment with bullet points, it shows the bullet points to the left of the other paragraphs. Also discussed [here](https://www.reddit.com/r/lbry/comments/dw5mx0/bullet_points_on_the_comments_seems_to_be_out_of/).

**To Reproduce**
1. Go to any video
2. Post a comment with bullet points, such as:
```
Example paragraph:

- Bullet point 1
- Another bullet point
```
3. After the comment is posted, it should show the bullet points to the left

**Expected behavior**
It should be inside the comment area or aligned with other paragraphs.

**Screenshots**
Here is [one](https://i.redd.it/dp8jr09wlly31.png).

Screenshot is from [this video](https://open.lbry.com/@elJefeDave:5/HolidayTurkeyNovember2019:9).

**System Configuration**

- LBRY Daemon version:0.43.3
- LBRY App version:0.37.1
- LBRY Installation ID:
- Operating system:Linux

**Additional context**
N/A

**Comment by DaniNz** [comment]
Hi, I just confirmed that the issue is still happening. Here's an screenshot.
![image](https://user-images.githubusercontent.com/16692007/103465827-07ca7b00-4d1e-11eb-8ddc-30c7fe0eae36.png)
Do you mind if I work in this issue?

**Comment by DaniNz** [comment]
Hi, I was practicing so this issue was good for me. I hope this is useful for you guys. 
https://github.com/lbryio/lbry-desktop/pull/5262

**Comment by DispatchCommit** [comment]
resolved by #5262
